_services/design-ux.md

```diff
@@ -4,13 +4,14 @@ title: Design / UX
 slug: design-ux
 weight: 3
 summary: |
-  Creating a beautiful design is a start. To design with purpose, aware of the user's every hope and dream, focused on usability that empowers content editors to soar: now that's another thing. UX in every permutation is what drives our design choices.
+  We design with purpose, guided by organizational goals and an acute awareness for all interaction stakeholders. We empower site admins to be more productive while crafting a beautiful, pleasant user journey.
 icon: "design-ux-icon.svg"
 vertical_image: "/assets/img/services/design-ux-hero.jpg"
 vertical_image_alt: "User experience design concepts"
 quote_project: "omega-md"
-services: [User Experience Strategy, Wireframing & Prototyping, Interface Design, Conversion Optimization, Concepting & Ideation, E-Commerce Web Design, Styleguide Design, Usability & User Testing]
+services: [User Experience Strategy, Concepting & Ideation, Wireframing & Prototyping, Style Guide Design, Conversion Optimization, Usability & User Testing]
 
 ---
 
-Through conceptualizing, UX design, and iterative wireframing, we design with mindfulness relevant to our clients' audiences and goals. We provide flexible, maintainable, and responsive custom designs using the best of the myriad web frameworks to our advantage, enabling us to prototype and build quickly. When it is the right fit, we partner with a design agency with whom we have an established relationship.
\ No newline at end of file
+Through rapid prototyping, leveraging the best of the modern web, our design process is iterative and translates efficiently to the final product. Driven by developing a deep understanding of the goals of our partners and their audiences, we craft flexible, usable, future-considerate responsive designs that empower content editors, are fit for all devices, and are a pleasure to use.
```

Comment:
I'd change to:

> Through rapid prototyping with the best modern tools, our design process...

This isn't just to remove "leveraging," I promise - I think "the best of the modern web" needs to be more specific and the phrasing needs some work.